cs/test/comm/Epoxy/EpoxyListenerTests.cs

```diff
@@ -120,6 +122,25 @@ public class EpoxyListenerTests : EpoxyTestBase
             Assert.AreEqual(connection.LocalEndPoint, disconnectedConnection.RemoteEndPoint);
         }
 
+        [Test]
+        public async Task OneConnectionStalledDuringHandshake_CanAcceptAnother()
+        {
+            EpoxyTransport transport = MakeTransport();
+            listener = transport.MakeListener(localhostEndpoint);
+            await listener.StartAsync();
+
+            var noHandshakeConnection = new TcpClient();
+            // This will just establish a TCP connection. It won't perform the Epoxy handshake, so
+            // the listener will just be sitting there waiting for the client to send some data.
+            await noHandshakeConnection.ConnectAsync(localhostEndpoint.Address, localhostEndpoint.Port);
+
+            var connectTask = transport.ConnectToAsync(localhostAddress);
+            bool didConnect = connectTask.Wait(TimeSpan.FromSeconds(30));
```

Comment:
This timeout is just to make the test fail way before the CI system's hour limit to get faster feedback. I can make it 10 seconds, but we do need to be careful about making it too aggressive, as the unit tests will be running on highly-loaded machines.